helm/wrongsecrets-ctf-party/templates/wrongsecrets-balancer/deployment.yaml

```diff
@@ -88,18 +66,10 @@ spec:
                   name: wrongsecrets-balancer-secret
                   key: metricsBasicAuthPassword
             {{- end }}
+          {{- with .Values.balancer.containerSecurityContext }}
           securityContext:
-            allowPrivilegeEscalation: false
-            readOnlyRootFilesystem: true
-            runAsNonRoot: true
-            capabilities:
-              drop:
-                - ALL
-              add:
-                - CAP_NET_ADMIN
-                - CAP_NET_BIND_SERVICE
-            seccompProfile:
-              type: RuntimeDefault
```

Comment:
See above?

Reply:
Yes, it is also present here like them:
https://github.com/OWASP/wrongsecrets-ctf-party/pull/249/files#diff-0e79142774e4871080c08cade4299253e8b84c26fab30421555d6a454406abf7R303